src/utils/component/index.ts

```diff
@@ -1,4 +1,4 @@
-import { ComponentDefinition as ComponentDefinitionTemplate } from "@prismatic-io/spectral";
+import { Component as ComponentDefinitionTemplate } from "@prismatic-io/spectral/dist/serverTypes";
```

Comment:
This was errantly referencing the `ComponentDefinition` prior to running through the convert layer. `prism` handles the result of spectral's `convertComponent`/`convertIntegration`.